.circleci/config.yml

```diff
@@ -84,9 +84,9 @@ jobs: # a collection of steps
           name: "Publish Release on GitHub"
           command: |
             # Retrieve version
-            VERSION=$(ls ./gradleBuild/distributions/Roddy-*.zip | sed -r 's/.*Roddy-([[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+(-.+)?).zip/\1/')
```

Comment:
Note that the test-suffix is not captured. The version comparison below only works for the pure version number.